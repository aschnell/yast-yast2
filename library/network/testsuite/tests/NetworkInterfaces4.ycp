{
// bug 72164

include "testsuite.ycp";

map READ = $[
    "network" : $[
	"section" : $[
	    "eth4"	: nil,
	],
	"value" : $[
	    "eth4"	: $["BOOTPROTO":"dhcp", "NAME": "we like 'singles'"],
	]
    ],
    "probe" : $[
	"system" : [],
    ],
    "target" : $[
	"tmpdir" : "/tmp",
    ],
];

TESTSUITE_INIT([READ], nil);
import "NetworkInterfaces";

DUMP("NetworkInterfaces::Read");
TEST(``(NetworkInterfaces::Read()), [READ], nil);
NetworkInterfaces::OriginalDevices = nil;

DUMP("NetworkInterfaces::Write");
TEST(``(NetworkInterfaces::Write("")), [READ], nil);
}
