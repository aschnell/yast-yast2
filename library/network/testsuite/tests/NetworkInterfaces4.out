Dump	NetworkInterfaces::Read
Dir	.network.section: ["eth4"]
Dir	.network.value."eth4": ["BOOTPROTO", "NAME"]
Read	.network.value."eth4".BOOTPROTO "dhcp"
Read	.network.value."eth4".NAME "single quotes aren't good"
Return	true
Dump	NetworkInterfaces::Write
Write	.network.value."eth4".BOOTPROTO "dhcp" true
Write	.network.value."eth4".NAME "single quotes arent good" true
Write	.network.value."eth4".STARTMODE "" true
Write	.network nil true
Return	true
